norfair/video.py

```diff
@@ -4,6 +4,7 @@
 import cv2
 from rich import print
 from rich.progress import BarColumn, Progress, TimeRemainingColumn
+from norfair import lib_metrics
```

Comment:
That module should be called `metrics` instead of `lib_metrics`.

Reply:
ok